ui-frontend/packages/catalog-ui-search/src/main/webapp/component/location-old/location-old.js

```diff
@@ -101,12 +112,21 @@ module.exports = Backbone.AssociatedModel.extend({
       value = options
     }
     Backbone.AssociatedModel.prototype.set.call(this, key, value, options)
-    Common.queueExecution(() => {
-      this.trigger('change', Object.keys(key))
-    })
   },
 
   initialize() {
+    this.listenTo(
```

Comment:
📝  Setup onChange listeners for the following events:
change:line change:polygon: When a user draws on the map, update the other position formats
change:dmsPointArray: When a user manually types in DMS, update the map and other position formats 
etc.